lib/Dialect/AIE/Transforms/AIEObjectFifoStatefulTransform.cpp

```diff
@@ -103,18 +115,28 @@ class DMAChannelAnalysis {
 
   /// Given a tile and DMAChannelDir, returns next usable channel index for
   /// that tile.
-  int getDMAChannelIndex(TileOp tileOp, DMAChannelDir dir) {
+  int getDMAChannelIndex(TileOp tileOp, DMAChannelDir dir, bool hasCrossTileIssue) {
```

Comment:
Is this an issue or a capability?

Reply:
Perhaps this is more descriptive:

```suggestion
  int getDMAChannelIndex(TileOp tileOp, DMAChannelDir dir, bool requiresAdjacentTileAccessChannels) {
```